### DISCUSSION TITLE
Alarm support for Microsoft teams webhook

### DISCUSSION THREAD

**Discussion by muthumdu2k8** [apache/skywalking#10299] [Q&A] [answered]
i created webhook url in slack and followed the ShyWalking document and it is working fine and able to get message from skywalking to slack.

In the same way i have created Microsoft teams webhook url. and it is not clear that how to implement in alarm-settings.yml. can anyone help me how to configure alarm-settings.yml with teams webhook url.

**Comment by pg-yang** [comment]
Hello, please refer to

https://github.com/apache/skywalking/blob/f7162d589863b9f909096fb089bb907387fc74f8/dist-material/config-examples/alarm-settings.yml#L57

https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-alarm/#webhook

**Comment by wu-sheng** [comment, marked as answer]
There isn't Microsoft teams webhook yet, you could try to implement it refer to existing hooks.
Contribution is welcome.

**Reply by muthumdu2k8** [reply]
thanks for you reply. sorry for late reply. I will try to work on this.

**Comment by keremerdem** [comment]
> i created webhook url in slack and followed the ShyWalking document and it is working fine and able to get message from skywalking to slack.

Hello, can you please share the steps? Sky is working with Docker and I didn't set Slack webhook.
